### DISCUSSION TITLE
When will achievement badges show on mobile?

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#28754] [Mobile] [answered]
How long do you think this the badge will come to come on mobile?

Edit: they did, thanks GitHub

**Comment by vansopa** [comment]
> How long do you think this function will take to come on mobile?

**Comment by alcere** [comment, marked as answer]
Hi @Nit-nit, sadly right now it is not part of our backlog. I am sharing your feedback with the mobile team and it will be evaluated. Thanks again, stay tuned!

**Reply by NitinIsFit** [reply]
Thanks for giving me your time bro, but seriously I can be ui/ux designer at GitHub to make their job easier 🤪🤪🤪
